src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -149,48 +183,12 @@ $pf-v6--include-toolbar-breakpoints: true !default;
   // }
 }
 
-// - Toolbar content - Toolbar content section - Toolbar expandable content
-.#{$toolbar}__content,
-.#{$toolbar}__content-section,
-.#{$toolbar}__expandable-content {
-  row-gap: var(--#{$toolbar}__content--RowGap);
-  column-gap: var(--#{$toolbar}--ColumnGap);
-}
-
-// - Toolbar content - Toolbar expandable content
-.#{$toolbar}__content,
-.#{$toolbar}__expandable-content {
-  padding-block-start: var(--#{$toolbar}__content--PaddingBlockStart, var(--#{$toolbar}__content--PaddingBlock));
-  padding-block-end: var(--#{$toolbar}__content--PaddingBlockEnd, var(--#{$toolbar}__content--PaddingBlock));
-  padding-inline-start: var(--#{$toolbar}__content--PaddingInlineStart, var(--#{$toolbar}__content--PaddingInline));
-  padding-inline-end: var(--#{$toolbar}__content--PaddingInlineEnd, var(--#{$toolbar}__content--PaddingInline));
-}
```

Comment:
These `row/column-gap`, `padding` settings were separated and given their own variables.